SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Android/Crashes.cs

```diff
@@ -86,9 +86,9 @@ static Task<bool> PlatformHasReceivedMemoryWarningInLastSessionAsync()
             return Task.Run(() => (bool)future.Get());
         }
 
-        static void PlatformTrackError(Exception exception, IDictionary<string, string> properties)
+        static void PlatformTrackError(Exception exception, IDictionary<string, string> properties, params ErrorAttachmentLog[] attachments)
```

Comment:
```suggestion
        static void PlatformTrackError(Exception exception, IDictionary<string, string> properties, ErrorAttachmentLog[] attachments)
```

The optional parameters needs to be only in the public layer.